app/src/main/java/org/mozilla/fenix/search/SearchFragmentStore.kt

```diff
@@ -56,6 +56,11 @@ sealed class SearchEngineSource {
      * Search engine for history
      */
     data class History(override val searchEngine: SearchEngine) : SearchEngineSource()
+
+    /**
+     * Search engine for bookmark
```

Comment:
Maybe we should change everything to the plural "bookmarks"

Reply:
Sure, I'll update